.gitignore

```diff
@@ -2,5 +2,10 @@ db.sqlite3
 
 __pycache__/
 *.pyc
+*.mo
 
 node_modules
+
+static/*
+!static/fonts
+!static/img
```

Comment:
I don't think we want to move the static files to `STATIC_ROOT` (`static`). I think we can leave them where they are, and `collectstatic` will move them here each time.

Reply:
Done in https://github.com/open-contracting/data-registry/pull/170/commits/c790ee71d5bb4171497a1cac6791327828d71e73